jest-test-mode.el

```diff
@@ -75,6 +75,27 @@
   :type '(list)
   :group 'jest-test-mode)
 
+(defcustom jest-test-command-string
+  "npx %s jest %s %s"
+  "The command by which jest is run.
+
+Placeholders are:
+
+1. Jest options (`jest-test-npx-options')
+2. Jest test options (`jest-test-options')
+3. The file name"
+  :initialize 'custom-initialize-default
+  :type 'string
+  :group 'jest-test-mode)
+
+(defcustom jest-test-use-relative-file-paths
+  nil
+  "When non-nil, test files are passed as relative paths from the project root."
+  :initialize 'custom-initialize-default
+  :type 'boolean
+  :safe #'booleanp
+  :group 'jest-test-mode)
+
```

Comment:
> Regarding path relativization: another way to think about it is that relative paths should always work, so it could be the default behavior (instead of adding a useless defcustom).

I think you're right -- I think relativizing by default is safe to do and should not affect any existing users. Let's remove this option and make the default behaviour to relativize.

Reply:
Thanks, I've removed the option and made it the default behavior.